cpplint.py

```diff
@@ -6334,9 +6376,25 @@ def main():
     _cpplint_state.PrintErrorCounts()
   finally:
     sys.stderr = backup_err
+=======
+
+  # Change stderr to write with replacement characters so we don't die
+  # if we try to print something containing non-ASCII characters.
+  if not Py3k:
+      sys.stderr = codecs.StreamReaderWriter(sys.stderr,
+                                             codecs.getreader('utf8'),
+                                             codecs.getwriter('utf8'),
+                                             'replace')
+
+  _cpplint_state.ResetErrorCounts()
+  for filename in filenames:
+    ProcessFile(filename, _cpplint_state.verbose_level)
+  _cpplint_state.PrintErrorCounts()
+>>>>>>> Added support for Python3000
```

Comment:
* I backup and restore sys.stderr (mostly in case someone uses cpplint as library, I think), you don't
* You use codecs.getreader('utf8'), codecs.getwriter('utf8')